format-specs/geoparquet.md

```diff
@@ -78,6 +79,16 @@ This is the binary format that the geometry is encoded in. The string 'WKB' to r
 [Well Known Binary](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry#Well-known_binary) is the only current option, but future versions
 of the spec may support alternative encodings. This should be the ["standard"](https://libgeos.org/specifications/wkb/#standard-wkb) WKB representation.
 
+#### bbox
+
+Bounding boxes are used to help define the spatial extent of each geometry column.
+Implementations of this schema may choose to use those bounding boxes to filter
+partitions (files) of a partitioned dataset.
```

Comment:
I think that could be nice to have, but I think it's also ok to ship 0.1 without fully describing how to do the spatial partitioning. I feel like partitioning is not the only reason to have a bbox - it's nice to be able to get the bounds without iterating through every feature. I see this as just establishing the field, and in 0.2 we'll tackle spatial indexing as a whole, with #13.

Reply:
A description like that would be useful, but doesn't necessarily belong here in the `bbox` specification, I think (especially if we want to go a bit more in depth). 
The main reason I mentioned it here was give _some_ context of why it could be useful. But I could also leave it out alltogether?

> 2\. and the bounding box of the root / common_metadata file would be the unary union of the bounding boxes (right?).

That's not something that is currently guaranteed, at least not in the spec (this is just about the bbox field for a single parquet file), and we also don't do that in the geopandas implementation actually. But so that's a good question ..

Given that a proper spatial partitioning is quite complex (eg what to do with the `_metadata` file if present), and we might want to promote other ways than a bbox in each individual file (that is not very efficient as you need to read the footer of each file to build up your spatial partitioning structure), I would indeed leave that for a separate discussion (and for 0.2), and focus here just on adding a bbox for individual files (and as @cholmes says, that can indeed be useful in general, not only for spatial partitioning.

Reply:
Indeed. Let's keep it simple for now. A hint as to why it might be useful, like you have now, is better than nothing IMO.